project/profiles/capacity4more/modules/c4m/user/c4m_user_notifications/includes/c4m_user_notifications_handlers.inc

```diff
@@ -202,17 +202,28 @@ function c4m_user_notifications_existing_group_content_updated($node) {
 /**
  * Email notification to inactive users.
  *
- * Loads inactive users and sends notification once a week, until activated.
+ * Loads inactive users and notifies once per interval, until user is activated.
+ * Invoked once an hour by cron.
  */
 function c4m_user_notifications_notify_inactive_users() {
   $interval = variable_get('c4m_inactive_users_reminder_interval', 7);
```

Comment:
@anvmn 
Change this to `c4m_inactive_users_reminder_days_interval`, plus add a comment above.